alert-common/src/main/java/com/synopsys/integration/alert/common/workflow/task/ScheduledTask.java

```diff
@@ -44,6 +44,7 @@
     public static final String STOP_SCHEDULE_EXPRESSION = "";
     //Spring Cron documentation  https://riptutorial.com/spring/example/21209/cron-expression
     public static final String EVERY_MINUTE_CRON_EXPRESSION = "0 0/1 * 1/1 * *";
+    public static final String EVERY_HOUR_CRON_EXPRESSION = "0 * * * * *";
```

Comment:
I don't think this is every hour, this looks like every minute. Have you tested this? If you start Alert you can go to the task management section and see the next run time to confirm if this is working as expected

Reply:
What cron standard are you using?

Reply:
The comment on line 45 references the Spring cron documentation that is used